dump/src/util/dump/Dump.java

```diff
@@ -1186,6 +1201,20 @@ private void renameFile( File currentFile, File newFile ) throws IOException {
       _log.info("renamed file " + currentFile + " to " + newFile);
    }
 
+   private void resetMeta() {
+      _sequence = (long)(Math.random() * 1000000);
```

Comment:
Shouldn't we use something better like `System.currentTimeMillis()` here?

Reply:
No idea, really. It was just some kind of (hopefully) non-zero random initializer, didn't touch any of that.

Reply:
Might as well use System.nanoTime()

Reply:
incrementation should get the job done at all times... fixed and pushed